src/editor/action/mouse/mouse_action.cpp

```diff
@@ -106,7 +106,7 @@ std::unique_ptr<editor_action> mouse_action::key_event(
 	|| event.key.keysym.sym == SDLK_DELETE) {
 		int res = event.key.keysym.sym - '0';
 		if (res > gamemap::MAX_PLAYERS || event.key.keysym.sym == SDLK_DELETE) res = 0;
-		const std::string* old_id = disp.map().is_starting_position(previous_move_hex_);
+		const std::string* old_id = disp.map().is_special_location(previous_move_hex_);
```

Comment:
Hmm, I'm questioning whether these changes of `is_starting_position` -> `is_special_location` are correct. They kind of look like they're expecting a side's starting location rather than a generic location ID? The `is_starting_position` function still exists, too...